lib/Ohmbrewer_Relay.cpp

```diff
@@ -118,12 +172,31 @@ bool Ohmbrewer::Relay::isOff() const {
 
 /**
  * Performs the Equipment's current task. Expect to use this during loop().
+ * Switches the power pin on or off for a generic relay, subclasses can utilize the controlPin
  * This function is called by work().
  * @returns The time taken to run the method
  */
 int Ohmbrewer::Relay::doWork() {
-    // TODO: Implement Relay::doWork
-    return -1;
+    int startTime = millis();
+    bool relayState = getState();
```

Comment:
There isn't any advantage to creating local variables for these things you're using getters for.